machine_learning/linear_discriminant_analysis.py

```diff
@@ -222,28 +244,44 @@ def main():
         for i in range(n_classes):
             while True:
                 try:
-                    user_count = int(input("Enter The number of instances for class_{}: ".format(i + 1)))
+                    user_count = int(
+                        input(
+                            "Enter The number of instances for class_{}: ".format(i + 1)
+                        )
+                    )
                     if user_count > 0:
                         counts.append(user_count)
                         break
                     else:
-                        print("Your entered value is {}, Number of instances should be positive!".format(user_count))
+                        print(
+                            "Your entered value is {}, Number of instances should be positive!".format(
+                                user_count
+                            )
+                        )
                         continue
                 except ValueError:
                     print("Your entered value is not numerical!")
 
         print("-" * 100)
 
-        user_means = []  # An empty list to store values of user-entered means of classes
+        user_means = (
+            []
+        )  # An empty list to store values of user-entered means of classes
```

Comment:
Put this comment on a separate line above so that black does not make your code ugly when it reformats.